core/src/main/java/hudson/cli/CLICommand.java

```diff
@@ -251,6 +255,10 @@ public int main(List<String> args, Locale locale, InputStream stdin, PrintStream
             stderr.println("Bad Credentials. Search the server log for "+id+" for more details.");
             return -1;
         } catch (Exception e) {
+            final String errorMsg = String.format("Unexpected exception occurred while performing %s command: %s",
+                    getName(), e.getMessage());
```

Comment:
If I am not mistaken the exception message will be printed by `printStackTrace`. No need to do it twice.